docs/join-the-network/node-types.md

```diff
@@ -82,3 +82,35 @@ network.
 
 The RPC node serves as a gateway for developers and users to interact
 with the Avail network without the need to run a full node locally.
+
+### Bootstrap nodes
```

Comment:
Bcoz this is relevant for both Substrate and LC networks, maybe we should add a line that both networks require this type of node?

Reply:
Went for a wider description there, that is applicable in pretty much all general cases, not only our networks